include/rmm/mr/device/logging_resource_adaptor.hpp

```diff
@@ -255,7 +261,7 @@ class logging_resource_adaptor final : public device_memory_resource {
 
   Upstream* upstream_;  ///< The upstream resource used for satisfying
                         ///< allocation requests
-};
+};                      // namespace mr
```

Comment:
I think this is the end of `class logging_resource_adaptor`; the end of `namespace mr` is at the bottom of the file.

Reply:
clang-format should be automating this comment.

Reply:
That's strange; is clang-format parsing something incorrectly? The end of `namespace mr` should be here https://github.com/rapidsai/rmm/blob/0caaa277f20cff1663b5aab89b006e7f8c6ecc79/include/rmm/mr/device/logging_resource_adaptor.hpp#L297

Reply:
The comment was incorrect. It added the `namespace mr` when there was a syntax error from a missing brace, and then when I corrected the syntax error the namespace comment doesn't get auto-removed.